lib/packages/shared-types/opensearch/main/transforms/legacy-transforms/temporary-extension.ts

```diff
@@ -3,9 +3,11 @@ import { events } from "shared-types";
 
 export const transform = () => {
   return events["legacy-event"].legacyEventSchema.transform((data) => {
-    const cleanData = omit(data, ["statusDate", "state", "parentId"]);
+    const cleanData = omit(data, ["parentId"]);
```

Comment:
I'm hoping we can wean ourselves off of lodash. This can be achieved with native JS:

```ts
const { _parentId, ...restOfData } = data;
```